### PR TITLE
ENH: Support vega and theta for BSEuropeanOption

### PR DESCRIPTION
**Pull request by masanorihirano**
#413

### CODE REVIEW

**Comment by masanorihirano** [comment]
#415

File: pfhedge/nn/modules/bs/european.py [hunk 1]
```diff
@@ -148,6 +148,61 @@ def gamma(
...
+        s, t, v = broadcast_all(log_moneyness, time_to_maturity, volatility)
+        price = self.strike * s.exp()
+        vega = npdf(d1(s, t, v)) * price * t.sqrt()
+
+        return vega
+
+    def theta(
+        self, log_moneyness: Tensor, time_to_maturity: Tensor, volatility: Tensor
+    ) -> Tensor:
+        """Returns theta (yearly) of the derivative.
```
Comment (simaki) [inline pfhedge/nn/modules/bs/european.py:180]: just a remark (not requiring any correction): unit of theta is not a priori; determined by that of volatility and time_to_maturity

File: tests/nn/bs/test_european.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import math
+
```
Comment (simaki) [inline tests/nn/bs/test_european.py:2]: nit:

```suggestion
```

**Comment by simaki** [comment]
The above comments are just nit things that don't really need corrections. Will merge in this Fri or so

**Comment by simaki** [comment]
closing as a duplicate of https://github.com/pfnet-research/pfhedge/pull/424

**Pull request closed without merging**